<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PtExploitDetectorGUIClass</class>
 <widget class="QMainWindow" name="PtExploitDetectorGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>685</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>685</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>685</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PtExploitDetector</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../PtExploitDetector/res/icon.ico</normaloff>../PtExploitDetector/res/icon.ico</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>660</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>660</height>
    </size>
   </property>
   <widget class="QLabel" name="labelTitle">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>300</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 16pt &quot;Open Sans&quot;;
	qproperty-alignment: 'AlignVCenter | AlignHCenter';
	color: #bdc3c7;
	background-color : #34495e;
}</string>
    </property>
    <property name="text">
     <string>PTEXPLOITDETECTOR</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="filePathPanel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	font: 10pt &quot;Consolas&quot;;
	qproperty-alignment: 'AlignTop | AlignLeft';
	border: 1px solid black;
	color: black;
	background-color: white;
}</string>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonOpen">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 8pt &quot;Open Sans&quot;;
	border: 0px;
	color: #bdc3c7;
	background-color: #2c3e50;
}
QPushButton:pressed {
	font: 8pt &quot;Open Sans&quot;;
	border: 0px;
	color: #bdc3c7;
	background-color: #34495e;
}</string>
    </property>
    <property name="text">
     <string>OPEN</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonAnalyze">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 8pt &quot;Open Sans&quot;;
	border: 0px;
	color: #bdc3c7;
	background-color: #2c3e50;
}
QPushButton:pressed {
	font: 8pt &quot;Open Sans&quot;;
	border: 0px;
	color: #bdc3c7;
	background-color: #34495e;
}</string>
    </property>
    <property name="text">
     <string>ANALYZE</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDump">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 8pt &quot;Open Sans&quot;;
	border: 0px;
	color: #bdc3c7;
	background-color: #2c3e50;
}
QPushButton:pressed {
	font: 8pt &quot;Open Sans&quot;;
	border: 0px;
	color: #bdc3c7;
	background-color: #34495e;
}</string>
    </property>
    <property name="text">
     <string>DUMP</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="exploitDataPanel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>300</width>
      <height>500</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>500</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPlainTextEdit {
	font: 10pt &quot;Consolas&quot;;
	qproperty-alignment: 'AlignTop | AlignLeft';
	border: 1px solid black;
	color: black;
	background-color: white;
}</string>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>exploitDataPanel</zorder>
   <zorder>labelTitle</zorder>
   <zorder>filePathPanel</zorder>
   <zorder>buttonOpen</zorder>
   <zorder>buttonAnalyze</zorder>
   <zorder>buttonDump</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PtExploitDetectorGUI.qrc"/>
 </resources>
 <connections/>
</ui>
